permissions/plugin-aws-secrets-manager-credentials-provider.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "aws-secrets-manager-credentials-provider"
+github: "jenkinsci/aws-secrets-manager-credentials-provider-plugin"
+paths:
+- "io/jenkins/plugins/aws-secrets-manager-credentials-provider"
```

Comment:
Your path is incorrect, since you are not providing your own <groupId> outside the parent pom, then you inherit the groupId from the parent pom which is org.jenkins-ci.plugins, so your path would need to be `org/jenkins-ci/plugins/aws-secrets-manager-credentials-provider`

Reply:
In the end I followed the example of warnings-ng-plugin and set my plugin’s groupId to io.jenkins.plugins, so this should now be valid.